app/migrations/Version20230927055621.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Mautic\CoreBundle\Doctrine\AbstractMauticMigration;
+
+final class Version20230927055621 extends AbstractMauticMigration
+{
+    public function up(Schema $schema): void
+    {
+        $table = $schema->getTable($this->prefix.'emails');
+
+        if (!$table->hasColumn('continue_sending')) {
+            $this->addSql("ALTER TABLE {$this->prefix}emails ADD `continue_sending` TINYINT(1) DEFAULT 0");
+            $this->addSql("UPDATE {$this->prefix}emails SET `continue_sending`= 1 WHERE publish_up IS NOT NULL");
```

Comment:
Isn't this a BC break? It doesn't send emails to the contacts that were added after the segment email was sent. right?

Reply:
This migration ensures all emails behave consistently, as in previous versions.
Probably I can add condition on type = list (exclude campaign emails)